components/eventing-controller/pkg/subscriptionmanager/beb/beb.go

```diff
@@ -6,7 +6,9 @@ import (
 	"strings"
 	"time"
 
+	beb2 "github.com/kyma-project/kyma/components/eventing-controller/pkg/handlers/beb"
```

Comment:
(nit) can we use better name than `beb2`?

Reply:
done